### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Maxwell-Yang-PDFTron**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -0,0 +1,1444 @@
...
+RNPdftron.encryptDocument("/sdcard/Download/new.pdf", "1111", "").then(() => {
+  console.log("done password");
+});
+```
+
+## DocumentView - Props
+
+A React component for displaying documents of different types such as PDF, docx, pptx, xlsx and various image formats.
+
+### Document
```
Comment (sgong-pdftron) [inline API.md:120]: Open a document

File: API.md [hunk 2]
```diff
@@ -0,0 +1,1444 @@
...
+
+```js
+<DocumentView
+  onDocumentError = {(error) => { 
+    console.log('Error occured during document opening:', error); 
+  }}
+/>
+```
+
+### Button / Tool
```
Comment (sgong-pdftron) [inline API.md:211]: UI Customization

File: API.md [hunk 3]
```diff
@@ -0,0 +1,1444 @@
...
+
+```js
+<DocumentView
+  onToolChanged = {({previousTool, tool}) => {
+    console.log('Tool has been changed from', previousTool, 'to', tool);
+  }}
+/>
+```
+
+### Leading Navigation Button
```
Comment (sgong-pdftron) [inline API.md:255]: part of UI customization

File: API.md [hunk 4]
```diff
@@ -0,0 +1,1444 @@
...
+
+```js
+<DocumentView
+  onZoomChanged = {(zoom) => {
+    console.log('Current zoom ratio is', zoom); 
+  }}
+/>
+```
+
+### Toolbar
```
Comment (sgong-pdftron) [inline API.md:427]: "Toolbar customization" move up to be close to UI customization section

File: API.md [hunk 5]
```diff
@@ -0,0 +1,1444 @@
...
+  onLongPressMenuPress = {({longPressMenu, longPressText}) => {
+    console.log('Long press menu item', longPressMenu, 'has been pressed');
+    if (longPressText !== '') {
+      console.log('The selected text is', longPressText);
+    }
+  }}
+/>
+```
+
+### Behavior
```
Comment (sgong-pdftron) [inline API.md:647]: "Custom Behavior"

File: API.md [hunk 6]
```diff
@@ -0,0 +1,1444 @@
...
+
+```js
+<DocumentView
+  collabEnabled={true}
+  currentUser={'Pdftron'}
+  currentUserName={'Hello_World'}
+/>
+```
+
+### Annotation (Form Field)
```
Comment (sgong-pdftron) [inline API.md:753]: "Annotations"

File: API.md [hunk 7]
```diff
@@ -0,0 +1,1444 @@
...
+--- | --- | ---
+filePath | string | the location of the saved document
+
+```js
+this._viewer.saveDocument().then((filePath) => {
+  console.log('saveDocument:', filePath);
+});
+```
+
+### Tool
```
Comment (sgong-pdftron) [inline API.md:1056]: "Annotation tools"

File: API.md [hunk 8]
```diff
@@ -0,0 +1,1444 @@
...
+
+```js
+this._viewer.getPageCropBox(1).then((cropBox) => {
+  console.log('bottom-left coordinate:', cropBox.x1, cropBox.y1);
+  console.log('top-right coordinate:', cropBox.x2, cropBox.y2);
+  console.log('width and height:', cropBox.width, cropBox.height);
+});
+```
+
+### Annotation (Form Field)
```
Comment (sgong-pdftron) [inline API.md:1157]: "Import/export annotations"

File: API.md [hunk 9]
```diff
@@ -0,0 +1,1444 @@
...
+
+```js
+// annotList is an array of annotation data in the format {id: string, pageNumber: int}
+const annotations = [{id: 'annot1', pageNumber: 1}, {id: 'annot2', pageNumber: 3}];
+this._viewer.exportAnnotations({annotList: annotations}).then((xfdf) => {
+  console.log('XFDF for 2 specified annotations', xfdf);
+});
+```
+
+#### flattenAnnotations
```
Comment (sgong-pdftron) [inline API.md:1228]: Create another "Annotations" section, move these out to there

File: API.md [hunk 10]
```diff
@@ -0,0 +1,1444 @@
...
+  'textField1': 'Test',
+  'textField2': 1234,
+  'checkboxField1': true,
+  'checkboxField2': false,
+  'radioButton1': 'Yes',
+  'radioButton2': 'No'
+});
+```
+
+### handleBackButton
```
Comment (sgong-pdftron) [inline API.md:1398]: New heading "Navigation"

**Review by sgong-pdftron — APPROVED** [review]

**Review by JamesTron — APPROVED** [review]

**Pull request merged**